src/js/services.js

```diff
@@ -1,47 +1,61 @@
 import React, {Component} from 'react';
 import ReactDOM from 'react-dom';
+import promise from 'es6-promise';
+import fetch from 'isomorphic-fetch';
+import 'babel-polyfill';
+import _ from 'lodash';
+
+const monthNames = ["Jan", "Feb", "March", "April", "May", "June", "July", "Aug", "Sept", "Oct", "Nov", "Dec"];
+let timelineArray;
 
 const renderedTimeline = (rawData) => {		
-	const monthNames = ["Jan", "Feb", "March", "April", "May", "June", "July", "Aug", "Sept", "Oct", "Nov", "Dec"];
-	let timelineArray = [];
-	for(let i in rawData) {
-		let tweetObj = rawData[i];
-		let date = new Date(tweetObj.createdAt);
+	timelineArray = [];
+	rawData.forEach( function(tweetObj, index) {
+		renderTweetObj(tweetObj, index);
+		console.log(tweetObj.message);
```

Comment:
you left a console log here. imagine everything you push is planned for production.